btree/btree.c

```diff
@@ -8562,6 +8578,61 @@ M0_INTERNAL int64_t m0_btree_lrulist_purge(int64_t size)
 	m0_rwlock_write_unlock(&list_lock);
 	return total_size;
 }
+
+/**
+ * Checks whether lru list purging is required based on used space
+ * watermark(enum m0_btree_used_space_watermark).
+ *
+ * @param user purge user
+ * @param size requested memory to be freed.
+ *
+ * @return int total freed memory.
+ */
+M0_INTERNAL int64_t m0_btree_lrulist_purge_check(enum m0_btree_purge_user user,
+						 int64_t size)
+{
+	int64_t size_to_purge;
+	int64_t purged_size = 0;
+
+	if (lru_space_used < M0_USW_LOW) {
```

Comment:
Use variables instead of enums so that they can be changed later as m0d command-line parameters. The default values for those variables could be the values set in the enums.
Make sure these variables are of global scope so that they can be changed from different source files when command-line is passed to m0d.

Reply:
Ok.